app/hotels/src/navigation/allHotels/navigationOptions/NavigationOptions.js

```diff
@@ -10,11 +10,18 @@ import HeaderRight from './HeaderRight';
 export default function getNavigationOptions() {
   return {
     headerStyle: {
-      height: 78,
-      marginTop: Platform.select({
+      height:
+        78 +
+        Platform.select({
```

Comment:
This could be in some const, because it's repetitive

Reply:
It could, I can fix 😊

Reply:
@jvaclavik 🚀